lib/appbundler/cli.rb

```diff
@@ -9,10 +9,12 @@ class CLI
     banner(<<-BANNER)
 * appbundler #{VERSION} *
 
-Usage: appbundler APPLICATION_DIR BINSTUB_DIR
+Usage: appbundler BUNDLE_DIR BINSTUB_DIR [GEM_NAME] [GEM_NAME] ...
```

Comment:
note that i'm pretty sure i changed appbundle-updater/kitchen-appbundle-updater to upgrade the appbundler in the shipped omnibus package.  you can't back that out or else slightly older omnibuses won't be able to grab master because they'll break on the bundle dependency.  our tests won't fail in travis because kitchen-appbundle-updater pulls the omnibus nightlies, but it'll mean that phil won't be able to test-bump a 12.6.0-ish install to master.